lean/models/click_options.py

```diff
@@ -82,7 +82,7 @@ def get_options_attributes(configuration: Configuration, default_lean_config_key
 
 
 def get_default_key(configuration: Configuration):
-    return "job-organization-id" if configuration.is_type_organization_id else configuration._id
+    return configuration._id
```

Comment:
hm tricky related to the other comment above I think, `job-organization-id` is what lean is expecting

Reply:
This one is not used anymore, there are no `OrganizationIdConfiguration` configurations anymore.